web/modules/custom/os2forms_selvbetjening_deprecations/src/Form/ElementsForm.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Drupal\os2forms_selvbetjening_deprecations\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\Url;
+use Drupal\webform\Plugin\WebformElementManagerInterface;
+use Drupal\webform\WebformEntityStorageInterface;
+use Drupal\webform\WebformInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Elements form.
+ */
+final class ElementsForm extends FormBase {
+  use StringTranslationTrait;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(
+    readonly private WebformElementManagerInterface $webformElementManager,
+    readonly private WebformEntityStorageInterface $webformEntityStorage,
```

Comment:
I think

```suggestion
    private readonly WebformElementManagerInterface $webformElementManager,
    private readonly WebformEntityStorageInterface $webformEntityStorage,
```

reads nicer …

Reply:
Fixed!